jsonnet/kube-prometheus/prometheus-adapter/prometheus-adapter.libsonnet

```diff
@@ -191,7 +191,7 @@ local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
       local rules =
         policyRule.new() +
         policyRule.withApiGroups(['metrics.k8s.io']) +
-        policyRule.withResources(['pods']) +
+        policyRule.withResources(['pods', 'nodes']) +
```

Comment:
didn't this also need `aggregate-to-cluster-reader` or is that an openshift thing?

Reply:
That is indeed an openshift thing, we need to augment that aggregation in CMO (after merging and pulling this in).

Reply:
ack